model/installation_database.go

```diff
@@ -25,9 +25,9 @@ const (
 	// hosted via Amazon RDS.
 	// TODO: update name value to aws-multitenant-rds-mysql
 	InstallationDatabaseMultiTenantRDSMySQL = "aws-multitenant-rds"
-	// InstallationDatabaseMultiTenantRDSPotgres is a PostgreSQL multitenant
+	// InstallationDatabaseMultiTenantRDSPostgres is a PostgreSQL multitenant
 	// database hosted via Amazon RDS.
-	InstallationDatabaseMultiTenantRDSPotgres = "aws-multitenant-rds-postgres"
```

Comment:
for some reason I find this typo amusing

Reply:
As do I!